lib/evmone/instructions_opcodes.hpp

```diff
@@ -152,6 +152,9 @@ enum Opcode
     OP_LOG3 = 0xa3,
     OP_LOG4 = 0xa4,
 
+    OP_DUPN = 0xb2,
+    OP_SWAPN = 0xb3,
```

Comment:
Let's go for 0xb3 + 0xb4 to match the EIP.

Reply:
Done

Reply:
Sorry to ask again, but to avoid conflicts with EIP-1153, let's use 0xb5 and 0xb6. The EIP has been updated.

Reply:
Done